loadData/formatJsonData.py

```diff
@@ -65,21 +59,22 @@ def parseSpoonacular(jsonData):
                        'gluten_free': data['glutenFree'],
                        'dairy_free': data['dairyFree'],
                        'peanut_free': 'UNKNOWN',
-                       'fats': nutrients['Fat']['percentOfDailyNeeds'],
-                       'saturated_fats': nutrients['Saturated Fat']['percentOfDailyNeeds'],
-                       'protein': nutrients['Protein']['percentOfDailyNeeds'],
-                       'cholesterol': nutrients['Cholesterol']['percentOfDailyNeeds'],
-                       'sugar': nutrients['Sugar']['percentOfDailyNeeds'],
-                       'sodium': nutrients['Sodium']['percentOfDailyNeeds']}
+                       'fats': [x['percentOfDailyNeeds'] for x in nutrients if x['name'] == 'Fat'][0],
```

Comment:
Nutrients are formatted in a weird way for Spoonacular. It lists the nutrition as a list of dictionaries like:
dicts = [{'name': 'Cholesterol', 'value': #}, {'name': 'Fat', 'value': #}] and I didn't know a cleaner way of doing it.